apis/core.oam.dev/v1alpha2/core_types.go

```diff
@@ -24,6 +24,13 @@ import (
 	runtimev1alpha1 "github.com/crossplane/crossplane-runtime/apis/core/v1alpha1"
 )
 
+// HELM defines the encapsulation in a Helm module.
+type HELM struct {
```

Comment:
Do we need to create two different struct for helm. Can we reuse line 301

Reply:
Fixed. We can reuser line 301.